src/libs/actions/User.js

```diff
@@ -174,6 +179,25 @@ function validateLogin(accountID, validateCode) {
     });
 }
 
+/**
+ * Checks if the expiresAt date of a user's ban is before right now
+ *
+ * @param {String} expiresAt
+ * @returns {boolean}
+ */
+function isBlockedFromConcierge(expiresAt) {
```

Comment:
This method pretty much just compares a date with `now`, could it be generalized and moved to DateUtils?

Reply:
Given that I'm just using the super simple one liner you gave below, i don't think we need to put it in dateUtils